webAO/master.js

```diff
@@ -173,6 +177,7 @@ function onMessage(e) {
 				+ `<a class="button" href="client.html?mode=join&ip=${ipport}${asset}">Join</a></li>`;			
 		}
 		masterserver.close();
+		return;
```

Comment:
why did you write this commit

Reply:
return to monke

Reply:
being more serious about it:
i noticed there were more functions that didn't have a return despite being done with their work